CMakeLists.txt

```diff
@@ -399,6 +399,15 @@ if(BUILD_TESTS)
                                  secp256k1.host http_parser.host sss.host
     )
 
+    # add_unit_test(
```

Comment:
Turned off for now because of transient instability, which I suspect is caused by the lack of safety in metrics and endpoint lookups, to be fixed separately.